README.md

```diff
@@ -52,6 +52,12 @@ To pull the latest code and update Refinery installation:
 $ fab vm update
 ```
 
+To run integration tests, hit "Run All Tests" in the Cypress UI after this command:
+```bash
+$ ./node_modules/.bin/cypress run --config baseUrl=http://192.168.50.50:8000
+```
+The results of integration tests can be viewed on the [Cypress Dashboard](https://dashboard.cypress.io/#/projects/odo3mu/runs).
+
```

Comment:
The purpose of README is just to provide quick start instructions on how to run the project. This should be moved to the testing wiki pages. Also, info on how to install Cypress on the host is missing.